src/components/login.js

```diff
@@ -1,7 +1,132 @@
-function Login(){
-    return(
-        <div>Login</div>
-    )
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import '../App.css'
+
+function Login() {
+  const navigate = useNavigate()
+  const [inputData, setInputData] = useState({
+    email: '',
+    password: ''
+  })
+  const [Error, setError] = useState({})
+
+  const validation = () => {
+    const error = {};
+
+    if (!inputData.email.trim()) {
+      error.Email = "Email is required";
+    }
+
+    if (!inputData.password.trim()) {
+      error.Password = "Password is required";
+    }
+    setError(error);
+  };
+
+  const handleChange = (e) => {
+    setInputData({ ...inputData, [e.target.name]: e.target.value })
+
+  }
+
+  const loginData = (e) => {
+    e.preventDefault();
+
+    validation();
+
+    if (!inputData.email.trim() || !inputData.password.trim()) {
+      return
+    }
+    axios.post("http://localhost:3000/auth/login", inputData)
+      .then((res) => {
+        if (res.status === 200) {
+          localStorage.setItem('authToken', res.data.token);
+          axios.post("http://localhost:3000/user/getEmail", { "email": inputData.email })
+            .then((res) => {
```

Comment:
What is the purpose of this request. Are we doing anything with the response that we are receiving after the getEmail api call?